puppet/zulip/manifests/postgres_appdb_base.pp

```diff
@@ -12,10 +12,21 @@
         # Needed for our full text search system
         "${postgresql}-tsearch-extras",
       ]
-      zulip::safepackage {
+      $debian_release_version_array = scanf("${::operatingsystemrelease} ", '%d')
+      $debian_release_version = $debian_release_version_array[0]
+      if $debian_release_version < 10 {
+        zulip::safepackage {
         $appdb_packages:
           ensure  => 'installed',
           require => Exec['setup_apt_repo'],
+        }
+      }
+      else {
+        exec {'build_tsearch_extras':
+          require => Package[$zulip::postgres_common::postgresql_dev],
+          command => "bash -c ${::zulip_scripts_path}/lib/build-tsearch-extras",
```

Comment:
`bash -c` is unnecessary.

What’s the story here? Did the apt repository stop supporting Debian 10?

Reply:
The apt repository is maintained by us; we just need to do a build for it.  I can give you the materials to do it @andersk.

Reply:
Till then I'm temporarily using this if-else block; if we can get the build done and published, then we can remove this if-else block altogether which would be a lot cleaner.

Reply:
Can you add a comment about how this is a temporary hack until we can get `tsearch_extras` included in `packagecloud`?